src/main/java/org/akip/service/dto/TaskDefinitionDTO.java

```diff
@@ -0,0 +1,189 @@
+        this.candidateUsers = candidateUsers;
+    }
+
+    public String getCandidateGroups() {
+        return candidateGroups;
+    }
+
+    public void setCandidateGroups(String candidateGroups) {
+        this.candidateGroups = candidateGroups;
+    }
+
+    public Map<String, String> getProps() {
+        return props;
+    }
+
+    public void setProps(Map<String, String> props) {
+        this.props = props;
+    }
+
+    public String getTaskId() {
+        return taskId;
+    }
+
+    public void setTaskId(String taskId) {
+        this.taskId = taskId;
+    }
+
+    public Integer getPriority() {
+        return priority;
+    }
+
+    public void setPriority(Integer priority) {
+        this.priority = priority;
+    }
+
+    public Map<String, String> getFormFields() {
+        return formFields;
+    }
+
+    public void setFormFields(Map<String, String> formFields) {
+        this.formFields = formFields;
+    }
+
+    public String getBpmnProcessDefinitionId() {
+        return bpmnProcessDefinitionId;
+    }
+
+    public void setBpmnProcessDefinitionId(String bpmnProcessDefinitionId) {
+        this.bpmnProcessDefinitionId = bpmnProcessDefinitionId;
+    }
+
+    public String getFormBuilder() {
+        return formBuilder;
+    }
+
+    public void setFormBuilder(String formBuilder) {
+        this.formBuilder = formBuilder;
+    }
+
+    public String getFormVersion() {
+        return formVersion;
+    }
+
+    public void setFormVersion(String formVersion) {
+        this.formVersion = formVersion;
+    }
+
+    public String getFormSchema() {
+        return formSchema;
+    }
+
+    public void setFormSchema(String formSchema) {
+        this.formSchema = formSchema;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        TaskDefinitionDTO that = (TaskDefinitionDTO) o;
+        return Objects.equals(id, that.id) && Objects.equals(taskId, that.taskId) && Objects.equals(name, that.name) && Objects.equals(documentation, that.documentation) && Objects.equals(assignee, that.assignee) && Objects.equals(candidateUsers, that.candidateUsers) && Objects.equals(candidateGroups, that.candidateGroups) && Objects.equals(priority, that.priority) && Objects.equals(formBuilder, that.formBuilder) && Objects.equals(formVersion, that.formVersion) && Objects.equals(formSchema, that.formSchema) && Objects.equals(props, that.props) && Objects.equals(formFields, that.formFields) && Objects.equals(bpmnProcessDefinitionId, that.bpmnProcessDefinitionId);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id, taskId, name, documentation, assignee, candidateUsers, candidateGroups, priority, formBuilder, formVersion, formSchema, props, formFields, bpmnProcessDefinitionId);
+    }
+
+    @Override
+    public String toString() {
+        return "TaskDefinitionDTO{" +
+                "id=" + id +
```

Comment:
Use only the id, name and taskId fields.

Reply:
Done!